guardian/memsub-promotions
# Fix Promo Tool's Google Group check

## What does this change?


The Google Group checking is not working in PROD. This PR fixes that. I needed to upgrade Node a bit (not all the way) to get things to run locally. This app needs a lot more TLC on the front end, including a not-in-group failure page, which will be addressed in future PRs.

## How to test

1. Find a colleague, ensure they are not in the required Google Group.
2. Test current PROD deployment that they have access (they shouldn't as they are not in the group).
3. Deploy this build to CODE.
4. Check they now don't have access.
5. Add them to the group.
6. Check their access returns.



## Have we considered potential risks?


Might need to eagerly add people who were using the tool to the group, and have an escalation process for people that will be missed.
- [x]  Various known people added to group
